cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -923,6 +927,66 @@ public KafkaBrokerConfigurationBuilder withTieredStorage(String clusterName, Tie
         return this;
     }
 
+    /**
+     * Configures the quotas based on the type of the plugin - {@link QuotasPluginKafka}, {@link QuotasPluginStrimzi}
+     *
+     * @param clusterName    Name of the cluster
+     * @param quotasPlugin   Configuration of the quotas plugin
+     * @return  Returns the builder instance
+     */
+    public KafkaBrokerConfigurationBuilder withQuotas(String clusterName, QuotasPlugin quotasPlugin) {
+        if (quotasPlugin != null) {
+            // for the built-in Kafka quotas plugin we don't need to configure anything
+            if (quotasPlugin instanceof QuotasPluginStrimzi quotasPluginStrimzi) {
+                printSectionHeader("Quotas configuration");
+
+                configureQuotasPluginStrimzi(clusterName, quotasPluginStrimzi);
+
+                writer.println();
+            }
+        }
+
+        return this;
+    }
+
+    /**
+     * Configures Strimzi quotas plugin
+     *
+     * @param clusterName           Name of the cluster
+     * @param quotasPluginStrimzi   Strimzi quotas plugin configuration
+     */
+    private void configureQuotasPluginStrimzi(String clusterName, QuotasPluginStrimzi quotasPluginStrimzi) {
+        // add TopicOperator's and CruiseControl's user to the excluded principals
+        List<String> excludedPrincipals = new ArrayList<>(List.of(
+            String.format("User:CN=%s,O=io.strimzi", KafkaResources.kafkaComponentName(clusterName)),
+            String.format("User:CN=%s,O=io.strimzi", CruiseControlResources.componentName(clusterName))
```

Comment:
This is wrong?

Reply:
Why it is wrong? I thought that the CC user should be there.

Reply:
It does not seem to be the same as what is used in authorization for super users?

Reply:
I used the `componentName` method, because it does exactly same thing as we have in the authorization:
```
String.format("User:CN=%s-%s,O=io.strimzi", clusterName, "cruise-control")
```
```
    public static String componentName(String clusterName) {
        return clusterName + "-cruise-control";
    }
```

or am I missing something and I don't see some typo?